### DISCUSSION TITLE
Polycom mibs

### DISCUSSION THREAD

**Discussion by bacus99** [glpi-project/glpi-agent#788] [Ideas]
### Your idea

Please add Polycom mibs in sysobjects.ids in the next version
Line 5525

2684	Polycom	NETWORKING
2684.1	Polycom	NETWORKING	polycomVideo
2684.1.1	Polycom	NETWORKING      polycomViewStation
2684.1.2        Polycom NETWORKING      polycomAudio
2684.1.3        Polycom NETWORKING      polycomData

**Comment by g-bougard** [comment]
Hi @bacus99 

the related project is here: https://github.com/glpi-project/sysobject.ids

You can even submit your own PR.
